### ISSUE TITLE
Regarding the provided labels for AbdomenCTCT

### ISSUE THREAD

**Issue by xiaogengen-007** [multimodallearning/convexAdam#16]
Hi,

Thank you so much for the great work and sharing the code!

I have a quick question regarding the labels shared at:

> For the former all infered train/test segmentations for the Learn2Reg tasks can be obtained at https://cloud.imi.uni-luebeck.de/s/cgXJfjDZNNgKRZe

In AbdomenCTCT_predict there are two folders, predictsTr and predictsTs, both containing 50 segmentations. What is the difference between them?

Thank you!

**Comment by xiaogengen-007** [comment]
Hi,

I quickly checked and found that predictsTr and predictsTs are the same. Not sure if they are duplicated by mistake.
Also, I found that nnU-Net outputs 14 labels, including an extra label for the duodenum compared to the original learn2reg dataset.

I have another naive question regarding the usage of segmentation labels from nnU-Net as "features" to optimize the registration. Since we are using Dice as the primary accuracy surrogate, it seems unfair to compare it with intensity driven methods (e.g. the ones using only NCC, MSE or MIND as loss functions). I admit that this also applies to weakly supervised learning methods as well, if we add label matching in the loss function, we are almost certain to get higher Dice scores. I guess another way to put my question is that is this approach considered a way to incorporate the Dice loss used in weakly supervised learning method to optimization based approaches?
 
I would love any of your thoughts on this matter.

Thanks!

**Comment by mattiaspaul** [comment]
Hi xiaogengen-007,

thanks for your interesting comments, yes the predictsTr/Ts folder are all produced by the nnUNet so contain the same segmentations, which are fairly close to the ground truth but may contain other structures as well. 
You raise a valid point that using nnUNet outputs as features together with Dice for evaluation is not ideal - yet many other weakly supervised method will be trained as well to focus mainly on those semantic structures. To the best of my knowledge one of the best unsupervised results for AbdomenCT (~50% Dice vs >70% Dice with label supervision) was reached by our recent MICCAI paper with cyclic self-training https://github.com/multimodallearning/reg-cyclical-self-train. It does outperform pure optimisation based registration.
Best wishes, Mattias
